sympy/printing/pretty/pretty_symbology.py

```diff
@@ -166,6 +166,9 @@ def xstr(*args):
 # latin subscripts
 for l in 'aeioruvx':
     sub[l] = LSUB(l)
+# subscripts from ftp://ftp.unicode.org/Public/6.3.0/ucd/UnicodeData.txt
+for n, l in enumerate('hklmnpst'):
```

Comment:
Perhaps these letters can just be added to the string in line 167: 'aeioruvx' -> 'aeioruvxhklmnpst' and the correct name will be supplied.